core/src/main/resources/hudson/model/UpdateCenter/DownloadJob/Skipped/status.jelly

```diff
@@ -0,0 +1,28 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```

Comment:
It is copy&pase, I would say it is correct in such a case. I do not feel to an author.